src/sass/_advantages.scss

```diff
@@ -0,0 +1,146 @@
+@use './variables' as var;
+@use './global';
+
+.advantages {
+  background-color: #fff5f6;
+
+  &__container {
+    @extend .container;
+    padding-top: 39px;
+    padding-left: 20px;
+
+    @media screen and (min-width: 768px) {
+      padding-top: 66px;
+      padding-left: 36px;
+    }
+
+    @media screen and (min-width: 1200px) {
```

Comment:
W tym projekcie punkt przerwania jest na 1280px, nie na 1200px. Zmień proszę we wszystkich @media.